src/styles/normalize.scss

```diff
@@ -0,0 +1,6 @@
+html {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
```

Comment:
as far as I remember, normalization means that we set default styles for a specific tag

check normalization.css https://necolas.github.io/normalize.css/8.0.1/normalize.css

and you set some value for the whole page
I believe you can place it in main.scss alongside with body {}

Reply:
well, in general so but if it's the whole task, I think it is possible to write down and so

Reply:
yeap, I've checked it 

basically, html is a tag
so it is a normalization of tag html